pkg/kubelet/kuberuntime/kuberuntime_manager_test.go

```diff
@@ -1494,246 +1493,6 @@ func TestComputePodActionsWithInitContainers(t *testing.T) {
 	}
 }
 
-func TestComputePodActionsWithInitContainersWithLegacySidecarContainers(t *testing.T) {
```

Comment:
I am surprised we need to delete all these tests. Do we have similar tests for other logic?

Reply:
There are the same tests for them.
`TestComputePodActionsWithInitContainers` and `TestComputePodActionsWithInitAndEphemeralContainers`